apps/aechannel/src/aesc_fsm.erl

```diff
@@ -3089,10 +3325,20 @@ init(#{opts := Opts0} = Arg) ->
                 _   -> StateInitF(Initiator)
             end,
     ClientMRef = erlang:monitor(process, Client),
+    BlockHashDelta =
+        case maps:get(block_hash_delta, Opts, use_defaults) of
```

Comment:
I'm with @uwiger in this case. You are using different default values when using `maps:get/3` to signal a missing value, which make it harder to comprehend. `find` has exactly this semantic. This applies to many cases in the code base.